forem/forem
# Fix Forem Ownership Field

## What type of PR is this? (check all applicable)


## Description
I mistakenly renamed a field from `forem_owner_secret` to `owner_secret`. This causes a failure when its expecting `forem_owner_secret` in the Registrations controller. 

This has no impact in production since. this view was behind a feature flag. 

## Related Tickets & Documents
https://github.com/forem/forem/issues/14647

## QA Instructions, Screenshots, Recordings
- Delete the database with `rails db:drop; rails db:create; rails db:migrate; rails data_updates:run`. 
- Enale the feature flag by. running `FeatureFlag.enable(:creator_onboarding)`
- Thereafter, go to the signup page, and try signing up.
- You signup should be successful. 

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

- [x] No, and this is why: https://github.com/orgs/forem/projects/36#card-68001436, I'll tackle this before we turn the feature flag on. 

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be updated. I have filled out the [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md) issue template so Community Success can help update the Admin Docs appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media.giphy.com/media/80TEu4wOBdPLG/giphy.gif?cid=ecf05e47jdmlljuk7jwrvrbet9v8zp2k3jyaubw03xljk0s4&rid=giphy.gif&ct=g)
